XYZDrivers/src/main/java/com/xyzdrivers/models/InsertClaim.java

```diff
@@ -0,0 +1,45 @@
+/**
+ * @file InsertClaim.java
+ * @author Nathan
+ * @created 02/11/17
+ * @modified 02/11/17
+ * @notes -
+ */
+
+package com.xyzdrivers.models;

```

Comment:
This is a service so should go in .services

Reply:
Yeah my bad, for some reason completely blanked that the model is basically just the data. I'll change the package name. Claim.java isn't a service though right? Where should that go?

Reply:
Claim.java is perfect where it is :)